app/Platform/Listeners/ResetPlayerProgress.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Listeners;
+
+use App\Platform\Actions\ResetPlayerProgressAction;
+use App\Site\Events\UserDeleted;
+use Illuminate\Broadcasting\InteractsWithSockets;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Events\Dispatchable;
+use Illuminate\Queue\SerializesModels;
+
+class ResetPlayerProgress implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithSockets;
+    use SerializesModels;
+
+    public function __construct(
+        public object $event,
+    ) {
+    }
+
+    public function handle(object $event): void
```

Comment:
I get this error trying to run the `delete-overdue-accounts` command:
```
jamiras@ubuntu22 pr/1864:~/source/RAWeb$ php artisan ra:site:user:delete-overdue-accounts
   Illuminate\Contracts\Container\BindingResolutionException 

  Unresolvable dependency resolving [Parameter #0 [ <required> object $event ]] in class App\Platform\Listeners\ResetPlayerProgress
  
    at vendor/laravel/framework/src/Illuminate/Container/Container.php:1118
    1114▕     protected function unresolvablePrimitive(ReflectionParameter $parameter)
    1115▕     {
    1116▕         $message = "Unresolvable dependency resolving [$parameter] in class {$parameter->getDeclaringClass()->getName()}";
    1117▕ 
  ➜ 1118▕         throw new BindingResolutionException($message);
    1119▕     }
    1120▕ 
    1121▕     /**
    1122▕      * Register a new before resolving callback for all types.

      +13 vendor frames 
  14  app/Site/Actions/ClearAccountDataAction.php:80
      App\Site\Events\UserDeleted::dispatch()

  15  app/Site/Commands/DeleteOverdueUserAccounts.php:39
      App\Site\Actions\ClearAccountDataAction::execute()
```

Reply:
Fixed in https://github.com/RetroAchievements/RAWeb/pull/1864/commits/f5c891cbaa1381531181721ad14e45773ef302ce